src/main/java/org/folio/rest/impl/InvoiceLineHelper.java

```diff
@@ -476,8 +483,11 @@ private Future<Void> updateInvoiceAndAffectedLines(ILProcessing ilProcessing, Li
     return persistInvoiceLines(invoice, lines, requestContext)
       .compose(v -> updateInvoice(ilProcessing, requestContext))
       .recover(t -> {
-        logger.error("Failed to update the invoice and other lines", t);
-        throw new HttpException(500, FAILED_TO_UPDATE_INVOICE_AND_OTHER_LINES.toError());
+        var param = new Parameter().withKey("invoiceId").withValue(invoice.getId());
+        var errorParam = new Parameter().withKey("errorMessage").withValue(t.getMessage());
+        var error = FAILED_TO_UPDATE_INVOICE_AND_OTHER_LINES.toError().withParameters(List.of(param, errorParam));
+        logger.error("Failed to update the invoice '{}' and other lines", invoice.getId(), t);
+        throw new HttpException(500, error);
```

Comment:
what do you think to introduce new method like this?

void throwHttpException(int httpCode, ErrorCode error, List<Parameter> params)

Reply:
and another method:

void throwHttpExceptionWithDetails(int httpCode, ErrorCode error, List params, Throwable t) - that will automatically add errorMessage param

Reply:
1. I also think about the idea, `var error..` is require for log in some places. Since JsonObject.mapFromObject(error) is not used in logs, we can use  `void throwHttpException(int httpCode, ErrorCode error, List params) ` this method is exits. I agree with it will be easier to find correct part with search
2.  void throwHttpExceptionWithDetails(int httpCode, ErrorCode error, List params, Throwable t) - that will automatically add errorMessage param I saw some problem with implementation of method, some unsupported exception while creating custom error message. I think it is okay to use first approach

Reply:
used void throwHttpException(int httpCode, ErrorCode error, List params)